api/models/company.py

```diff
@@ -8,3 +8,4 @@ class Company(BaseModel):
     location = models.TextField()
     slogan = models.TextField(blank=True, default='')
     logo_url = models.URLField(blank=True, default='')
+    email = models.EmailField(blank=True, default='')
```

Comment:
Done in https://github.com/countryrudd/api.devlink.tech/pull/20

For future reference: Please don't add a field to a model without making a migration because the ORM will freak out on non-existent columns when performing operations to a given model's object.

Reply:
There is most likely a merge conflict here, if there isn't, please remove the one you added.